raiden/tests/utils/blockchain.py

```diff
@@ -47,6 +47,8 @@ def wait_until_block(chain, block):
     curr_block = chain.block_number()
     while curr_block < block:
         curr_block = chain.next_block()
+    # ctx switch to allow background tasks to proceed:
+    gevent.sleep(1)
```

Comment:
I removed this again after changing the waiting model to `wait_until_block` + `event.wait(timeout`